framework/wazuh/agent.py

```diff
@@ -1053,15 +1026,13 @@ def get_outdated_agents(agent_list: list = None, offset: int = 0, limit: int = c
                                       some_msg='Some agents information was not returned',
                                       none_msg='No agent information was returned'
                                       )
+    # TODO: The GET /agents/outdated endpoint use case must be reviewed since the manager in 5.0 will not have wazuh-db
     if agent_list:
-        # Get manager version
-        manager = Agent(id='000')
-        manager.load_info_from_db()
 
         rbac_filters = get_rbac_filters(system_resources=get_agents_info(), permitted_resources=agent_list)
 
         with WazuhDBQueryAgents(offset=offset, limit=limit, sort=sort, search=search, select=select,
-                                query=f"version!={manager.version}" + (';' + q if q else ''),
+                                query=f"version!=v5.0.0" + (';' + q if q else ''),
```

Comment:
Perhaps we could get this value from the [\_\_version\_\_](https://github.com/wazuh/wazuh/blob/9ceffa5ea6e20df09d3b65a36ae10ee086983068/framework/wazuh/__init__.py#L20C1-L20C13) variable -> `from wazuh import __version__`